apps/src/utils.js

```diff
@@ -110,7 +110,9 @@ exports.stripQuotes = function(inputString) {
  * Defines an inheritance relationship between parent class and this class.
  */
 Function.prototype.inherits = function (parent) {
-  this.prototype = _.create(parent.prototype, { constructor: parent });
+  this.prototype = Object.create(parent.prototype);
+  this.prototype.constructor = this;
+  this.superPrototype = parent.prototype;
```

Comment:
If this is only used in BeeItemDrawer, I'm perhaps most inclined to not have superPrototype at all and set __base = DirtDrawer.prototype.
If we think this will be used elsewhere, googling around a little shows $super to be a semi-common approach (though I'm not sure I personally like it as I see the $ and think jquery).  Ultimately, I'm not sure I have any super strong opinions and will leave it up to you :)